indexing-hadoop/src/test/java/org/apache/druid/indexer/DeterminePartitionsJobTest.java

```diff
@@ -51,6 +52,11 @@
 @RunWith(Parameterized.class)
 public class DeterminePartitionsJobTest
 {
+  @Nullable
+  private static final Long NO_TARGET_ROWS_PER_SEGMENT = null;
```

Comment:
nit: this seems _less_ clear to me than just using `null` inline, especially since intellij displays parameter name for literal values by default.

Reply:
The named constant can be useful for tools that do not display the parameter names (e.g., diff tools like the github PR diff). Perhaps the name can be improved (e.g., DEFAULT_TARGET_ROWS_PER_SEGMENT)?